frontend/components/SocialShare.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+const SocialShare = ({
+  link = typeof window !== 'undefined'
+    ? window.location.href
+    : 'https://home2health.org.au/',
+  title = 'Home2Health',
+  text = title,
+}) => {
+  return (
+    <div className="container">
+      <strong>Share With</strong>
+      <a
+        className="btn-floating btn-small waves-effect waves-light light-blue darken-4"
+        target="_blank"
```

Comment:
You will want to add ref='noopener noreferrer' https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
cool, thanks

Reply:
@noobling `ref='noopener noreferrer' ` or `rel='noopener noreferrer' `

Reply:
It’s rel sorry